src/main/java/duke/exception/ErrorMessage.java

```diff
@@ -0,0 +1,18 @@
+package duke.exception;
+
+/**
+ * Stores error messages to be shown to the user.
+ */
+public class ErrorMessage {
```

Comment:
Would this be better if it was an Enum?

Reply:
Hi, you can take at look at adressbook-level2 [Messages.java](https://github.com/se-edu/addressbook-level2/blob/master/src/seedu/addressbook/common/Messages.java) which I referred to.